src/streaming/TextTracks.js

```diff
@@ -71,6 +72,8 @@ function TextTracks() {
         displayCCOnTop = false;
         topZIndex = 2147483647;
 
+        imsc1Parser = require('imsc');
```

Comment:
Typically, we use the ES6 import statement for dependencies. See round10 and codem-isoboxer examples. Is there any reason why this pattern should not be used here too?